support/helpers.rb

```diff
@@ -5,42 +5,65 @@
 require "yaml"
 require "webrick"
 
-FRAMEWORKS = {
-  "react" => {
+CONFIGS = {
+  "react/cra" => {
     :root => "build",
     :build_dir => "build",
     :js_dir => "static/js",
+    :packages => [
+      "@appsignal/react"
+    ],
+  },
+  "react/vite" => {
+    :root => "dist",
+    :build_dir => "dist",
+    :js_dir => "assets",
+    :packages => [
+      "@appsignal/react"
+    ],
   },
   "vue" => {
     :root => "dist",
     :build_dir => "dist",
-    :js_dir => "js"
+    :js_dir => "js",
+    :packages => [
+      "@appsignal/vue"
+    ],
   },
   "angular" => {
     :root => "dist/app",
     :build_dir => "dist",
-    :js_dir => "app"
+    :js_dir => "app",
+    :packages => [
+      "@appsignal/angular"
+    ],
   },
   "stimulus" => {
     :root => "public",
     :build_dir => "public",
-    :js_dir => ""
+    :js_dir => "",
+    :packages => [
+      "@appsignal/stimulus"
+    ],
   }
 }
 
 PACKAGE_MANAGER = "yarn"
 
 def all_apps
-  FRAMEWORKS.map do |language, config|
-    Dir["frameworks/#{language}/*"].sort
-  end.flatten.map do |path|
+  Dir["frameworks/*/*"].sort.map do |path|
```

Comment:
```suggestion
  Dir["frameworks/*/*"].map do |path|
```
I remember RuboCop telling us this isn't needed in other apps because Ruby sorts it by default.

Reply:
We also don't really care about the order here, as far as I can tell. 👍